docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -1708,6 +1708,12 @@ The `query_scheduler` block configures the query-scheduler.
 # CLI flag: -query-scheduler.additional-query-queue-dimensions-enabled
 [additional_query_queue_dimensions_enabled: <boolean> | default = false]
 
+# (experimental) Use an experimental version of the query queue which has the
+# same behavior as the existing queue, but integrates tenant selection into the
+# tree model. (default false)
```

Comment:
```suggestion
# (experimental) Use an experimental version of the query queue, which has the
# same behavior as the existing queue but integrates tenant selection into the
# tree model.
```

Reply:
I don't think we need to mention the default value in the comments if we already have it in the setting itself.